docs/architecture/adr-068-preblock.md

```diff
@@ -0,0 +1,60 @@
+# ADR 068: Preblock
+
+## Changelog
+
+* Sept 13, 2023: Initial Draft
+
+## Status
+
+DRAFT
+
+## Abstract
+
+Introduce `PreBlock`, which runs before begin blocker other modules, and allows to modify consensus parameters, and the changes are visible to the following state machine logics.
+
+## Context
```

Comment:
looks good, when we do the follow up pr for prefinalizeblock we can add that here. for this pr it looks good